internal/provider/resource_gitlab_project_issue.go

```diff
@@ -23,7 +23,7 @@ var _ = registerResource("gitlab_project_issue", func() *schema.Resource {
 
 -> During a terraform destroy this resource will close the issue. Set the delete_on_destroy flag to true to delete the issue instead of closing it.
 
-~> **Experimental**: while the base functionality of this resource works, it may be subject to minor change.
+~> **Experimental** while the base functionality of this resource works, it may be subject to minor change.
```

Comment:
<img width="695" alt="Screen Shot 2022-03-09 at 8 08 13 PM" src="https://user-images.githubusercontent.com/9969202/157586810-8c77b4b6-11ce-4577-8743-73b7bee42798.png">

```suggestion
~> **Experimental** While the base functionality of this resource works, it may be subject to minor change.
```